tools/cache.py

```diff
@@ -149,7 +149,6 @@ def get(self, shortname, creator, what=None, force=False):
           what = 'system asset'
       message = 'generating ' + what + ': ' + shortname + '... (this will be cached in "' + cachename + '" for subsequent builds)'
       logger.info(message)
-      self.ensure()
```

Comment:
this was not needed before this PR, correct, and is just a cleanup?

Reply:
Yes, I added to the PR description: 

"Also remove superfluous ensure() calls.  Since we call this during
 the constructor these days the cache can be assumed to always exist."